README.md

```diff
@@ -104,7 +104,8 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](#)
+- [Live Demo Link](https://smart-budget-aw9r.onrender.com/)
+- [Loom video link](https://www.loom.com/share/ea5e2bb4931f4a859cdb6821a938586b)
```

Comment:
- [ ] [REQUIRED] Excellent presentation @techEmprez 👍🏽 , however, as stated by the project requirements, kindly highlight a piece of the code that you are proud of making. This will enable reviewers to be double sure that you built this project y yourself.
Let’s fix this 👍🏽

Reply:
All issues have been resolved and changes adequately corrected.
Please, for the next reviewer, kindly reach out via zoom or slack if need be for clarification.
Thank you.